zeppelin-server/src/test/java/org/apache/zeppelin/service/NotebookServiceTest.java

```diff
@@ -263,6 +264,26 @@ public void testNoteOperations() throws IOException {
     assertEquals(2, notesInfo.size());
     verify(callback).onSuccess(notesInfo, context);
 
+    // test moving corrupted note to trash
+    Note corruptedNote = notebookService.createNote("/folder_1/corruptedNote", "test", true, context, callback);
+    String corruptedNotePath = notebookDir.getAbsolutePath() + corruptedNote.getPath() + "_" + corruptedNote.getId() + ".zpln";
+    // corrupt note
+    try {
+      FileWriter myWriter = new FileWriter(corruptedNotePath);
+      myWriter.write("{{{I'm corrupted;;;");
+      myWriter.close();
+      System.out.println("Successfully wrote to the file.");
```

Comment:
Please remove `System.out.println` and work with asserts
You should use [try-with-ressource](https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html) for the FileWriter object.

Reply:
Done

Reply:
Done, thx for the suggestions